src/server/game/Conditions/ConditionMgr.cpp

```diff
@@ -21,7 +21,13 @@
 // Can have CONDITION_SOURCE_TYPE_NONE && !mReferenceId if called from a special event (ie: eventAI)
 bool Condition::Meets(ConditionSourceInfo& sourceInfo)
 {
-    ASSERT(ConditionTarget < MAX_CONDITION_TARGETS);
+    // ASSERT(ConditionTarget < MAX_CONDITION_TARGETS);
+    if (ConditionTarget >= MAX_CONDITION_TARGETS)
+    {
+        LOG_DEBUG("condition", "ConditionTarget is greater or equal than MAX_CONDITION_TARGETS");
```

Comment:
```LOG_DEBUG("condition", "ConditionTarget %d for for condition (Entry: %u Type: %u Group: %u) is greater or equal than MAX_CONDITION_TARGETS", ConditionTarget, SourceEntry, SourceType, SourceGroup);```
And also change that to error instead of debug.